app/controllers/api/v1/passwords_controller.rb

```
@@ -0,0 +1,75 @@
+          )
+  
+          unless user
+            return render json: { error: "Invalid or expired token" }, status: :unprocessable_entity
+          end
+  
+          if params[:password].blank? || params[:password_confirmation].blank?
+            return render json: { error: "Password and confirmation are required" }, status: :unprocessable_entity
+          end
+  
+          if params[:password] != params[:password_confirmation]
+            return render json: { error: "Passwords do not match" }, status :unprocessable_entity
```

RuboCop (1.75.5)

[fatal] unexpected token tSYMBOL
(Using Ruby 3.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)